File: protocol/conformance/fixtures/ts/add.ts
// Conformance fixture for the typescript frontend's outcome shape (str-hy9b.A5).
// A trivially executable function so `execute` lands on the success path and
// the response carries `outcome.status = "completed"`.
export function add(a: number, b: number): number {
  return a + b;
}

File: protocol/conformance/fixtures/rust/add.rs
// Conformance fixture for the rust frontend's outcome shape (str-hy9b.A5).
// A trivially executable free function so `execute` lands on the success path
// and the response carries `outcome.status = "completed"`.
pub fn add(a: i64, b: i64) -> i64 {
    a + b
}
